widget/scroller_test.go

```diff
@@ -31,9 +31,13 @@ func TestScrollContainer_MinSize(t *testing.T) {
 	rect := canvas.NewRectangle(color.Black)
 	rect.SetMinSize(fyne.NewSize(500, 50))
 	scroll := NewScrollContainer(rect)
-	scroll.SetMinSize(fyne.NewSize(100, 100))
+	assert.Equal(t, fyne.NewSize(scrollContainerMinSize, scrollContainerMinSize), scroll.MinSize())
```

Comment:
Please don't use internal constants for test expectations. Use literals instead. As long as the constant is not exported it is an implementation detail.
For exported constants I would recommend to test the constant's value separately and test functions returning its value using the constant.

Reply:
That hadn't really crossed my mind, but happy to use it as a rule